### PR TITLE
📤 Change sent messages view to be ordered by -sent_on

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/msgs/views.py [hunk 1]
```diff
@@ -143,7 +144,10 @@ def get_queryset(self, **kwargs):
             last_90 = timezone.now() - timedelta(days=90)
             queryset = queryset.filter(created_on__gte=last_90)
 
-        return queryset.order_by("-created_on", "-id").distinct("created_on", "id")
+        if self.show_channel_logs:
+            queryset = queryset.prefetch_related("channel_logs")
```
Comment (rowanseymour) [inline temba/msgs/views.py:148]: unrelated change so that subclasses don't have to remember to do this

File: temba/msgs/views.py [hunk 2]
```diff
@@ -143,7 +144,10 @@ def get_queryset(self, **kwargs):
...
+
+        return queryset.distinct("created_on", "id")
```
Comment (rowanseymour) [inline temba/msgs/views.py:150]: need to remember why this is here and whether it's needed or when it's needed...

**Review by nicpottier — APPROVED** [review]
